packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -109,11 +109,10 @@ def test_vip(dcos_api_session, container: Container, vip_net: Network, proxy_net
     failure_stack = []
     for test in setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
         cmd, origin_app, proxy_app, proxy_net = test
-        log.info('ORIGIN APP INFO')
-        log.info(wait_for_tasks_healthy(dcos_api_session, origin_app))
-        log.info('PROXY APP INFO')
+        log.info('Deploying origin app')
+        wait_for_tasks_healthy(dcos_api_session, origin_app)
```

Comment:
I actually didn't intend to leave these statements in the first time I wrote this. They provide a large, hard-to-parse output and they only output that information when things are working smoothly. Thus, they were not even useful for debugging when I was actively working on it. In general, I think we should avoid logging output unless we have some idea of why we want to see it at that time. Or else we will be back where we were a few months ago where we logged the complete marathon app status every 2 seconds on all deploys, leading to logs so large that my browser would crash trying to view them.